content/introduction/_index.md

```diff
@@ -5,30 +5,45 @@ draft: false
 weight: 1
 ---
 
-{{<title>}} is a vendor-agnostic single distribution of libraries to provide observability for your microservices and monoliths alike. OpenCensus is an open source community developed project. Its origins stem from the rewrite of what has been Google's observability systems for the past 10 years.
-
-OpenCensus is implemented in a plethora of languages and it allows you to singly collect metrics and traces
-and then export them to a variety of backends.
+{{<title>}} makes getting critical telemetry out of your services easy and automatic. Its origins stem from the rewrite of what has been Google's observability systems for the past 10 years. OpenCensus currently provides libraries for a number of languages that allow you to capture, manipulate, and export metrics and distributed traces to the backend(s) of your choice.
 
 In this section we will walk through what OpenCensus is, what problems it solves, and how it can help your project.
 
 {{% children %}}
 
 Or, if you are ready to integrate OpenCensus in to your project, visit the [Quickstart](/quickstart).
 
-#### Partners & Contributors
```

Comment:
This is critical for community and brand association, why this removal?

Reply:
I'd originally removed it as it was already on the homepage, but after reviewing I agree that we should leave it

Reply:
... as someone who's a bit of an idiot when it comes to Git, what's the best way to revert this change?

Reply:
You can just recopy back from the original source https://raw.githubusercontent.com/census-instrumentation/opencensus-website/master/content/introduction/_index.md
otherwise you'd have to go through your git history

Reply:
Re-added